### DISCUSSION TITLE
Theming RNE components - styles not being applied

### DISCUSSION THREAD

**Discussion by ajp8164** [react-native-elements/react-native-elements#3595] [Q&A]
I guess this more of a question than an issue since I'm trying to create an implementation that follows the doc. I setup a theme with a Button as below. The style is not being applied. It seems simple enough but I must be missing something.

I logged `titleStyle` [here](https://github.com/react-native-elements/react-native-elements/blob/v4.0.0-rc.5/packages/base/src/Button/Button.tsx#L203) and it's `undefined`.

theme
```
export const theme = createTheme({
  Button: {
    titleStyle: {
      color: 'red',
    },
    disabledStyle: {
      backgroundColor: 'red',
    },
  },
...
```

App.tsx
```
import { theme } from 'theme'; // Defined above
...
  return (
    <ThemeProvider theme={theme}>
      <NavigationContainer linking={linking}>
        <Provider store={store}>
          <PersistGate loading={null} persistor={persistor}>
            <ColorModeSwitch>
              <ErrorBoundary onError={onError}>
                <MainNavigator />
              </ErrorBoundary>
            </ColorModeSwitch>
          </PersistGate>
        </Provider>
      </NavigationContainer>
    </ThemeProvider>
  );
```

In my screen
```
          <Button
            title={'my  title'}
          />
```

**Comment by theadultnoble** [comment]
What navigator is the app screen located at. Passing the theme that far down might not work

**Reply by ajp8164** [reply]
I gave up trying to get it to work. I'm just sticking with an exported global style sheet. I don't recall the exact nav location but the main nav encloses a tab navigator and I tested in a tab>stack view.
